Gruntfile.js

```diff
@@ -0,0 +1,86 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+
+    concat: {
+      options: {
+        separator: ';',
+      },
+      dist: {
+        src: [
+          'reservation-service/public/bundle.js',
+          'hero-photo-service/client/dist/bundle.js',
+          'Review-Service/client/dist/app.js',
+          'about-service/public/app.js'
+        ],
+        dest: 'public/bundle.js'
+      },
+      // dist: {
```

Comment:
Commented code.  Are you holding onto this for later?

Reply:
I will remove it for now, thanks for bringing my attention to it @burkegg